tasks/13.txt

```diff
@@ -1,3 +1,19 @@
 Create a SQL query to count current and discontinued products:
 
-| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
\ No newline at end of file
+| TotalOfCurrentProducts | TotalOfDiscontinuedProducts |
+
+var totalCountProd_Disc_0= db.getCollection('products').find({Discontinued:{$eq:0}}).count();
+var totalCountProd_Disc_1 = db.getCollection('products').find({Discontinued:{$eq:1}}).count();
+
+db.products.aggregate([
+    { "$group": {
+        _id: null,
+        "TotalOfCurrentProducts": {
+            $avg:totalCountProd_Disc_0
+        },
+        "TotalOfDiscontinuedProducts": {
+             $avg:totalCountProd_Disc_1
+        }
+    }},
+    {$project:{_id:0}}
+])
```

Comment:
last query (db.products.aggregate) does useless work. 
Also in the first queries you can use just count function with query.